index.html

```diff
@@ -33,18 +33,12 @@
              xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 100 100" preserveAspectRatio="xMidYMid"
              style="margin: 0 auto; background: none;">
           <g transform="translate(50,50)">
-            <circle cx="0" cy="0" r="12.5" fill="none" stroke="#2962ff" stroke-width="4"
+            <circle cx="0" cy="0" r="12.5" fill="none" stroke="#9C27B0" stroke-width="4"
                     stroke-dasharray="39.269908169872416 39.269908169872416" transform="rotate(227.921)">
               <animateTransform attributeName="transform" type="rotate" values="0 0 0;360 0 0" times="0;1"
```

Comment:
Should we replace the `animateTransform` element with css animation?

Reply:
Yes the svg animations are deprecated